src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -151,6 +151,36 @@ static uint16_t ep0_pending[2];                   // Index determines direction
 static uint16_t _allocated_fifo_words_tx;         // TX FIFO size in words (IN EPs)
 static bool _rx_ep_closed;                        // Flag to check if RX FIFO size needs an update (reduce its size)
 
+// Calculate the RX FIFO size according to recommendations from reference manual
+// ep_size in words
+static inline uint16_t calc_rx_ff_size(uint16_t ep_size)
```

Comment:
ep_size should be better in bytes. But this is minor,I could do it myself

Reply:
If you prever, i leave this up to you! :)